src/vaadin-context-menu.html

```diff
@@ -420,15 +423,25 @@
           if (listenOn && openOn) {
             this._listen(listenOn, openOn, this._boundOpen);
 
-            // note: these styles don't seem to work in Firefox on iOS.
-            listenOn.style.webkitTouchCallout = 'none';
-            listenOn.style.webkitUserSelect = 'none';
-
             this._oldListenOn = listenOn;
             this._oldOpenOn = openOn;
           }
         }
 
+        _setListenOnUserSelect(value) {
+          // note: these styles don't seem to work in Firefox on iOS.
+          this.listenOn.style.webkitTouchCallout = value;
+          this.listenOn.style.webkitUserSelect = value; // Chrome, Safari
+          this.listenOn.style.MozUserSelect = value; // Firefox
```

Comment:
This can be removed too, see above comment.

Reply:
Removed.